components/Meta.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import Head from 'next/head'
+
+const Meta = ({ title, keywords, description }) => {
+    return (
+        <Head>
+            <meta name='viewport' content='width=device-width, initial-scale=1'/> 
```

Comment:
We could also maybe add open graph meta, for example:

`<meta property="og:title" content="Colby Fayock - A UX Designer &amp; Front-end Developer Blog" />
<meta property="og:type" content="website" />
<meta property="og:image" content="/static/website-social-card-44070c4a901df708aa1563ac4bbe595a.jpg" />
<meta property="og:url" content="https://www.colbyfayock.com" />`

Check this page:
[https://www.freecodecamp.org/news/what-is-open-graph-and-how-can-i-use-it-for-my-website/](https://www.freecodecamp.org/news/what-is-open-graph-and-how-can-i-use-it-for-my-website/)

Reply:
> We could also maybe add open graph meta (Graph meta tags are snippets of code that control how URLs are displayed when shared on social media), for example:
> 
> `<meta property="og:title" content="Colby Fayock - A UX Designer &amp; Front-end Developer Blog" /> <meta property="og:type" content="website" /> <meta property="og:image" content="/static/website-social-card-44070c4a901df708aa1563ac4bbe595a.jpg" /> <meta property="og:url" content="https://www.colbyfayock.com" />`
> 
> Check this page: https://www.freecodecamp.org/news/what-is-open-graph-and-how-can-i-use-it-for-my-website/

I thought of that, but since there are different meta for og and twitter I thought we could add this closer to the end when we would have a more clear idea of what we need to display and how, as you can also choose the how with the twitter ones.

Reply:
> The indentation is off here. Let's keep it to 2 spaces on every file of the project.
> 
> I'll check for a linter package to beautify the code before commiting

Fixed. I've changed my tab to 2 spaces.

Reply:
Cool. Let's leave the rest of the meta for later on ;)